.gitignore

```diff
@@ -1,4 +1,4 @@
-# Ignore editor artefacts
+# Ignore editor artifacts
```

Comment:
Can we not get into an American English vs British English spelling war? Let's leave this one alone ;-)